Server/serverSlate.go

```diff
@@ -38,18 +37,19 @@ func main() {
 
 	hostReader := bufio.NewReader(conn)
 	timeStamps := make([]string, KEY_PRESS_COUNT+1)
+	keyPressSequence := make([]string, KEY_PRESS_COUNT+1)
 
 	message, _ := hostReader.ReadString('*') // using * as delim and hardcoded in app to append * at end of message
 	fmt.Println(message)
 	if message == "started capture"+"*" {
-		simulateKeyPress(KEY, KEY_PRESS_COUNT, timeStamps)
+		simulateKeyPress(KEY, KEY_PRESS_COUNT, timeStamps, keyPressSequence)
```

Comment:
The way this is being passed, it makes it seem like keyPressSequence is an input of this method, when it really is an output (?). Why not make the method create and return the slice instead?